.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "python.linting.pylintEnabled": false,
+  "python.linting.pycodestyleEnabled": true,
+  "python.linting.enabled": true
+}
```

Comment:
Did you commit this on purpose? Not sure if IDE settings should be tracked.

Reply:
Sorry, I will remove. I did `git add .` by accident